### ISSUE TITLE
EFES question regarding new repository

### ISSUE THREAD

**Issue by ajstaub** [EpiDoc/Tutorials#19]
Hello! I'm currently watching the Epidoc tutorial "Introduction to EFES Publication Tool". I'm trying to follow along in EFES so I can make sure I'm understanding, but when I go to "new repository", my screen does not match the screen shared by Irene. It asks me to change my server, rather than displaying Type/ID/Title. Is this something we won't be able to access/do correctly in full until after our meeting tomorrow as we work through EFES together, or am I doing something wrong (that is, should I be able to see Type/ID/Title)? I've attached an image of the screen I see on EFES.

Thank you!

<img width="736" alt="Screen Shot 2022-03-22 at 1 25 08 PM" src="https://user-images.githubusercontent.com/101992980/159539960-a15e8917-b18f-42f3-ab60-81fbfade4c0e.png">

**Comment by gabrielbodard** [comment]
I'm not sure I can troubleshoot this, but from your screenshot it looks like you have clicked "change" next to "Sesame server"? Try clicking on "New repository" in the left menu again, and see if you now have what you expect?

**Comment by IreneVagionakis** [comment]
I have seen this happening in the past when using EFES with a version of Java later than 8: you could try checking what version of Java you have and, if it is later than 8, downgrading it.

**Comment by ajstaub** [comment]
@gabrielbodard, unfortunately, when I go from Admin>components>sesame>admin, I come to this page (screenshotted above), and when I hit "New Repository" nothing new happens and I remain on that page. @IreneVagionakis I have just checked my java and it is 8; should I still downgrade it? Thank you!!

**Comment by emylonas** [comment]
@ajstaub are you onsite or remote? If onsite, we can try to look at this Wednesday at lunch or before lunch (12:30 or 1).
what do you see when you type `java -version` on the command line? I have the following, and am successful with EFES.
```
java version "1.8.0_181"
Java(TM) SE Runtime Environment (build 1.8.0_181-b13)
```
(that means that I'm running Java 8.181)
also when you start up EFES on the command line, do you see the following 3 lines of messages, or is there a long endless scroll? 

```
Buildfile: /Users/emylonas/Desktop/EFES-Dartmouth/EFES-master/local.build.xml
Development server is running at http://127.0.0.1:9999
Quit the server with CONTROL-C.
```

@gabrielbodard and @IreneVagionakis we had one install yesterday that produced what looked like error messages, but still displayed the EFES startup screen, so I didn't pursue further. 

No real thoughts, but some info to compare.

**Comment by gabrielbodard** [comment]
@emylonas Oh yes, there are lots of error messages in the commandline that you can usually ignore. (Good luck troubleshooting this tomorrow!)

**Comment by ajstaub** [comment]
@emylonas Unfortunately, I am remote! I did go to the Help session yesterday though, and even though I couldn't resolve the new repository issue, the person helping said that I didn't need to create a new repository to access the example files and make edits in our sessions today, so it should be alright! Thank you so much!

**Comment by gabrielbodard** [comment]
That's true, you only need the repository to index and search in EFES, you can transform and view your EpiDoc files on the web without it.
